terraform/environments/digital-prison-reporting/secrets.tf

```diff
@@ -14,6 +14,19 @@ resource "aws_secretsmanager_secret" "nomis" {
   )
 }
 
+# Nomis Source Secrets in format required by Athena Federated Query
+resource "aws_secretsmanager_secret" "nomis_athena_federated" {
```

Comment:
@harichintala1 This actually contains the same creds as the existing nomis secret but the connector requires them in a slightly different format unfortunately. I presume that when this is promoted to preprod and prod you would populate this placeholder secret with a value?

Reply:
We said I'd add a placeholder secret for this in terraform and then we can update it with the real credentials as it is promoted through the environments